pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -63,15 +64,18 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 	rootDisplay := fmt.Sprintf("%s (Operator-Version: \"%s\" Active-Plan: \"%s\")", instance.Name, instance.Spec.OperatorVersion.Name, lastPlanStatus.Name)
 	rootBranchName := tree.AddBranch(rootDisplay)
 
-	for name, plan := range ov.Spec.Plans {
-		if name == lastPlanStatus.Name {
-			var planDisplay string
+	plans := make([]string, 0, len(ov.Spec.Plans))
+	for k := range ov.Spec.Plans {
+		plans = append(plans, k)
+	}
```

Comment:
```suggestion
	plans := funk.Keys(ov.Spec.Plans)
```
https://github.com/thoas/go-funk#funkkeys

Reply:
Nice, I hadn't looked at go-funk before! 👍